src/main.cpp

```diff
@@ -176,8 +180,8 @@ MAKE_HOOK_MATCH(MainMenuViewController_DidActivate, &MainMenuViewController::Did
     //UnityEngine::GameObject* onlineButtonTextObj = onlineButton->get_transform()->Find(textName)->get_gameObject();
     
     // Find the GameObject and get the component for the online button's text
-    static auto* searchPath = il2cpp_utils::newcsstr<il2cpp_utils::CreationType::Manual>("MainContent/OnlineButton/Text");
-    TMPro::TextMeshProUGUI* onlineButtonText = self->get_gameObject()->get_transform()->Find(searchPath)->get_gameObject()->GetComponent<TMPro::TextMeshProUGUI*>();
+    //static auto* searchPath = il2cpp_utils::newcsstr<il2cpp_utils::CreationType::Manual>("MainContent/OnlineButton/Text");
+    TMPro::TextMeshProUGUI* onlineButtonText = self->get_gameObject()->get_transform()->Find("MainContent/OnlineButton/Text")->get_gameObject()->GetComponent<TMPro::TextMeshProUGUI*>();
```

Comment:
This could be wrapped in a `ConstString` instead of allocating on each pass, albeit with one more line of boilerplate.